build.gradle

```diff
@@ -10,26 +10,26 @@ buildscript {
 plugins {
```

Comment:
Just a heads up but if would it be possible to switch to maven? As fast as gradle is i'd like to maintain consistency between build tools used by solar. However, since this is a fork of a public repo it would also make sense that this uses gradle, and switching to maven would probably be quite a bit of work. Just some thoughts!

Reply:
In terms of the maven/gradle debate I tend to prefer maven as it's much more easily and widely understood even if gradle is faster.

Here though, the main concern is reducing the maintainability burden as we merge in changes from upstream Velocity. I don't see much benefit to switching to maven *unless possibly* it helps with management of deployment credentials.

Reply:
fair enough